notations.tex

```diff
@@ -1,4 +1,4 @@
-% !TEX root = zeth-protocol-specification.tex
+%!Tex root = ./zeth-protocol-specification.tex
```

Comment:
(I'm not familiar with these - are these two types interchangable?)
We now seem to have a mix of `% !TEX` and `%!Tex`.  Are they interchangeable?  Should we use one form consistently?

Reply:
On my machine !Tex causes errors and !TEX did not :(

Reply:
Thanks for the comments guys.

> We now seem to have a mix of % !TEX and %!Tex.

@dtebbs AFAICT this was already the case before this PR, see below for e.g.:
- https://github.com/clearmatics/zeth-specifications/blob/master/chap00-changelog.tex#L1
- https://github.com/clearmatics/zeth-specifications/blob/master/notations.tex#L1

but it seems indeed that this inconsistency needs to be corrected (see point below)

> On my machine !Tex causes errors and !TEX did not :(

@mpzajac, right I think these should be uppercase everywhere, as per:
- https://latex.org/forum/viewtopic.php?t=31920
- https://github.com/SublimeText/LaTeXTools#compiling-latex-files

I'll change back to that here and switch to uppercases everywhere else.

Reply:
So, as said above, these `%!TEX` comments have been set in a very inconsistent way throughout the entire source (some are set on the appendix files after a line break, some are forgotten etc etc).
As far as I can see, these comments are used by people compiling from an editor (as illustrated by the fact that the CI passes just fine), see: https://latextools.readthedocs.io/en/latest/settings/ (section: `Project-Specific Settings`) and https://latextools.readthedocs.io/en/latest/features/ (section: `Multi-file documents`). Seems like these directives are used to overwrite some configuration file values.

@mpzajac I'd be in favor to avoid editor specific directives in the source here, do you have a local config file? If these are really making a difference and make your life easier, and since these comments seem to be pretty widely used with UI editors (and are simply ignored when not using an IDE) can I let you update all these `%!TEX` comments appropriately in the source?

Reply:
As discussed on a call: I think that "Tex" vs "TEX" stuff is probably a linux vs unix stuff. (TEX should work everywhere), not an editor issue -- this only tells the latex engine where the root file is, when the engine is run on a non-root file.

Reply:
I did some tests locally using latexmk/pdflatex (on ubuntu and also on macOS) and these comments don't seem to be taken into consideration even when I try to compile a chapter individually. After a few more Google search, I found a few other links suggesting that these "magic comments" have to deal with the compilation behavior of the IDEs:
- https://stackoverflow.com/questions/46593261/latex-tex-code
- https://tex.stackexchange.com/questions/78101/when-and-why-should-i-use-tex-ts-program-and-tex-encoding/84687#84687

Anyway, I'm not using IDEs and compile the document from the Makefile, so I can't reproduce - and won't spend ages on that, so I'll add the `%!TEX root` comments everywhere (for consistency at least) and merge this PR. If you guys run into troubles to compile to doc later on, please open a PR and suggest fixes.

Reply:
I did some tests locally using latexmk/pdflatex (on ubuntu and also on macOS) and these comments don't seem to be taken into consideration even when I try to compile a chapter individually. After a few more Google search, I found a few other links suggesting that these "magic comments" have to deal with the compilation behavior of the IDEs:
- https://stackoverflow.com/questions/46593261/latex-tex-code
- https://tex.stackexchange.com/questions/78101/when-and-why-should-i-use-tex-ts-program-and-tex-encoding/84687#84687

Anyway, I'm not very familiar with the latex compiler and am not using IDEs (I simply compile the document from the Makefile), so I can't reproduce - and won't spend ages on that, so I'll add the `%!TEX root` comments everywhere (for consistency at least) and merge this PR. If you guys run into troubles to compile to doc later on, please open a PR and suggest fixes.